src/libs/language-python/api-parser/tests/modules/not_serializable.py

```diff
@@ -0,0 +1,16 @@
+""" A module with a dawson-valid Python function, but with an invalid config """
+
+def myfunction(event, context):
+    """ This function is dawson-valid """
+    print(event, context)
+    return "Hello world"
+
+def another_function():
+    """ Just a normal function """
+    pass
+
+# this configuration is not JSON-serializable
+myfunction.api = {
+    "path": "/test",
```

Comment:
small change needed here: path should not start with a slash (see https://dawson.sh/docs.html#4-function-configuration)